packages/compass-schema-validation/src/modules/validation.ts

```diff
@@ -113,6 +121,36 @@ export interface ValidationState extends Validation {
   prevValidation?: Validation;
 }
 
+export const VALIDATION_TEMPLATE = `/**
```

Comment:
Do you think it would maybe make sense to move this to mongodb-constants eventually with some other templates? I think search indexes are already there

Reply:
Eventually once we start to think of other consumers of this I think it makes sense for sure. Right now we're only using this here in Compass so I think it would add more complexity.

Reply:
FWIW we are the only consumer of the search templates too as far as I know 🙂 There is some benefit of having this consistently in one place, there are also some tests in the package to automatically validate the formatting for example